.gitattributes

```diff
@@ -39,3 +39,4 @@
 *.png binary
 *.jpg binary
 *.gif binary
+*.pdf binary
```

Comment:
This is fine to leave in as merging will sort it out but is redundant to this PR:
https://github.com/raft-tech/TANF-app/pull/1236